saleor/graphql/product/tests/test_product.py

```diff
@@ -5546,6 +5546,42 @@ def test_update_product_with_page_reference_attribute_value(
     assert product_type_page_reference_attribute.values.count() == values_count + 1
 
 
+def test_update_product_with_empty_input_collections(
+    product, permission_manage_products, staff_api_client
+):
+    # given
+    query = """
+mutation updateProduct($productId: ID!, $input: ProductInput!) {
+  productUpdate(id: $productId, input: $input) {
+    productErrors {
+      field
+      message
+      code
+    }
+    product {
+      id
+    }
+  }
+}
+
+    """
+    product_id = graphene.Node.to_global_id("Product", product.pk)
+    variables = {
+        "productId": product_id,
+        "input": {"collections": ""},
```

Comment:
Wasn't it `collections: [""]` that was failing (empty string, but inside the list)?